assets/js/renderer.js

```diff
@@ -383,6 +383,11 @@ $(document).ready(function () {
             if (fileNames === undefined) return;
             loadDatabaseFromFile(fileNames[0]);
         });
+    }
+
+    // Open a SQLite Database File
+    openSQLiteFileButton.click(function() {
+        openSQLite();
```

Comment:
You can pass in the function directly:

```
openSQLiteFileButton.click(function() {
    openSQLite();
})
```

Change to:

`openSQLiteFileButton.click(openSQLite);`